charts/kubeclarity/values.yaml

```diff
@@ -203,12 +207,29 @@ kubeclarity-postgresql:
   ## Setup database name and password
   existingSecret: kubeclarity-postgresql-secret
   postgresqlDatabase: kubeclarity
+  # secretKey: postgresql-password
 
-  ## Enable security context
+  serviceAccount:
+    enabled: true
+  securityContext:
+    # Default is true for K8s. Enabled needs to false for OpenShift restricted SCC and true for anyuid SCC
+    enabled: true
+    # fsGroup specification below is not applied if enabled=false. enabled=false is the required setting for OpenShift "restricted SCC" to work successfully.
+    fsGroup: 1001
   containerSecurityContext:
+    # Default is true for K8s. Enabled needs to false for OpenShift restricted SCC and true for anyuid SCC
     enabled: true
+    # runAsUser specification below is not applied if enabled=false. enabled=false is the required setting for OpenShift "restricted SCC" to work successfully.
     runAsUser: 1001
-    runAsNonRoot: true
+  volumePermissions:
+    enabled: true
```

Comment:
default is `enabled: false`

Reply:
Updated so this is `enabled: false` by default for K8s installs and added description for OpenShift.